### DISCUSSION TITLE
Add the ability to "unblur" the background during gameplay breaks

### DISCUSSION THREAD

**Discussion by iamminimoon** [ppy/osu#13103] [Ideas]
**Describe the new feature:**
Similar to "Lighten playfield during breaks", but this one undoes the blur effect set on the gameplay background to a certain percentage (0% blur would be my suggestion).

**Proposal designs of the feature:**
The feature could be added as a toggle right under the "Lighten playfield during breaks" toggle or, for clear differentiation between settings, have each toggle to be put under the background adjustment slider in correlation (e.g. playfield lighten toggle under the "Background dim" slider and background unblur toggle under the "Background blur" slider).

**Comment by bdach** [comment]
IMO if we want to go this way, the "lighten during breaks" feature should also remove blur. I don't think having two toggles is necessary or productive. I guess the name for that could be a bit problematic, maybe something like "Show storyboards during breaks"? "Disable background effects during breaks"?

**Comment by frenzibyte** [comment]
Changing blur countless times seems to be expensive as I've done it myself as a side-proposal and [got rejected](https://github.com/ppy/osu/pull/5639#issuecomment-519748719).

**Comment by peppy** [comment]
Blur has since been optimised quite heavily. It may be feasible to do now. I agree it should not be its own setting but be the default if it is decided it looks better.
